### ISSUE TITLE
Javascript error in Admin interface running 'SHOW CONTINUOUS QUERIES' 

### ISSUE THREAD

**Issue by nathanielc** [influxdb/influxdb#3820]
When you run `SHOW CONTINUOUS QUERIES` you get a js error `Uncaught TypeError: Cannot read property 'map' of undefined` https://github.com/influxdb/influxdb/blob/master/shared/admin/js/admin.js#L312

Looking at the JSON response from the server the queries are returned but js error prevents them from being displayed.

version 0.9.3

**Comment by chosenken** [comment]
It looks like in build 0.9.4.1 this is still an issue.  When I search for ```SHOW CONTINUOUS QUERIES``` I get the response ```Query returned no results!```,  however when I view the response I see the continuous query was returned as json.
```
{
  "results": [
    {
      "series": [
        {
          "name": "_internal",
          "columns": [
            "name",
            "query"
          ]
        },
        {
          "name": "mod_stats",
          "columns": [
            "name",
            "query"
          ],
          "values": [
            [
              "gobal_request_counts",
              "CREATE CONTINUOUS QUERY gobal_request_counts ON mod_stats BEGIN SELECT count(value) INTO \"mod_stats\".\"default\".\"global.1m\" FROM \"mod_stats\".\"default\".atlas_response_global GROUP BY time(1m) END"
            ]
          ]
        }
      ]
    }
  ]
}
```

**Comment by beckettsean** [comment]
Confirmed that 0.9.4.1 Admin UI appears to do nothing when submitting `SHOW CONTINUOUS QUERIES` but the result is indeed returned, just never displayed.

**Comment by jeanpralo** [comment]
I have the same problem using the nightly build: "0.9.5-nightly-ace0e03"

A call to the api works 

curl -G http://127.0.0.1:8086/query --data-urlencode “db=db" --data-urlencode "q=show continuous queries”

**Comment by nathanielc** [comment]
This is fixed as part of https://github.com/influxdb/influxdb/pull/4375

A few show queries would return a row with no values and caused a js error. A check is now in place.

**Comment by nathanielc** [comment]
Fixed by #4375
